api/test/integration/api-key-guard.e2e-spec.ts

```diff
@@ -23,23 +23,25 @@ describe('API Key Guard Tests', () => {
     await app.close();
   });
 
-  it('should succeed when correct header is present', async () => {
+  it('should error but because of unauthorized (got passed the pass key)', async () => {
```

Comment:
should we keep a succeed test?

Reply:
you right, I was being lazy with it. Updated the test to be logged in now and we look for a success